packages/api-bindings/src/lens/graphql/wallet.graphql

```diff
@@ -10,13 +10,18 @@ query OwnedHandles($for: EvmAddress!, $limit: LimitType, $cursor: Cursor) {
 }
 
 query ProfilesManaged(
-  $request: ProfilesManagedRequest!
+  $for: EvmAddress!
+  $includeOwned: Boolean
+  $limit: LimitType
+  $cursor: Cursor
   $profileCoverSize: ImageTransform = {}
   $profilePictureSize: ImageTransform = {}
   $activityOn: [AppId!]
   $fxRateFor: SupportedFiatType = USD
 ) {
-  result: profilesManaged(request: $request) {
+  result: profilesManaged(
+    request: { for: $for, includeOwned: $includeOwned, limit: $limit, cursor: $cursor }
```

Comment:
ah, perfect, didn't know this flag existed.

Reply:
I think the example code in (`LoginPage.tsx` and tsdoc of `useLogin` should mention this hook rather than `useProfiles`.